packages/widget/src/components/ChainSelect/ChainSelect.tsx

```diff
@@ -41,10 +41,12 @@ export const ChainSelect = memo(({ formType }: FormTypeProps) => {
     setCurrentChain,
   } = useChainSelect(formType)
 
-  const { isAllNetworks, setIsAllNetworks } = useChainOrderStore((state) => ({
-    isAllNetworks: state.isAllNetworks,
-    setIsAllNetworks: state.setIsAllNetworks,
-  }))
+  const { showAllNetworks, isAllNetworks, setIsAllNetworks } =
+    useChainOrderStore((state) => ({
+      showAllNetworks: state[`${formType}ShowAllNetworks`],
+      isAllNetworks: state[`${formType}IsAllNetworks`],
+      setIsAllNetworks: state.setIsAllNetworks,
+    }))
```

Comment:
Please use an array here instead of object, it's usually more efficient and the variable names can stay the same 🙏